deploy/distribute.py

```diff
@@ -22,7 +23,7 @@
 )
 @click.option(
     '--distributor-tx',
-    help='Distributor contract address.'
```

Comment:
what is distributor txid used for? I see that you use it to get the block number in which the contract has been deployed, but it isn't used anywhere else.

Reply:
Just remove it - `Distributor.distributor_block` isn't used anywhere so there's no point in keeping it.

btw unfortunately it's not possible to get block id using an address, there's lengthy discussion about that on geth's issue tracker. :)

Reply:
Ah, sorry. Yes. distributor tx hash is useless. I'll remove it.